pkg/skaffold/build/cluster/cluster.go

```diff
@@ -42,7 +47,47 @@ func (b *Builder) Build(ctx context.Context, out io.Writer, tags tag.ImageTags,
 		defer teardownDockerConfigSecret()
 	}
 
-	return build.InParallel(ctx, out, tags, artifacts, b.buildArtifactWithKaniko)
+	kanikoArtifacts, otherArtifacts := splitArtifacts(artifacts)
+
+	pArts, parallelErr := build.InParallel(ctx, out, tags, kanikoArtifacts, b.buildArtifactWithKaniko)
+	if parallelErr != nil {
+		logrus.Warnf("error building kaniko artifacts in parallel: %v", parallelErr)
+	}
+
+	sArts, err := build.InSequence(ctx, out, tags, otherArtifacts, b.runBuildForArtifact)
```

Comment:
why did we decide at `InSequence`? Maybe we should mention this in the docs if we have a good reason - but it might actually just work with `InParallel` as well - in the end the majority of the work is just spinning up pods on cluster, right?

Reply:
I think we did that in case one artifact depended on another -- but since we don't explicitly promise that artifacts will be built in order I think it's reasonable to just do `InParallel` for cluster builds.

Reply:
Yeah, I think we should stick to `InParallel`